gradio/routes.py

```diff
@@ -40,7 +40,7 @@ class ORJSONResponse(JSONResponse):
     media_type = "application/json"
 
     def render(self, content: Any) -> bytes:
-        return orjson.dumps(content, option=orjson.OPT_SERIALIZE_NUMPY)
+        return orjson.dumps(content)
```

Comment:
Don't we still need `OPT_SERIALIZE_NUMPY` in case the variable value is numpy? 

https://github.com/gradio-app/gradio/pull/1606/

Reply:
You're right, just fixed.